.github/docker/linux/x86/Dockerfile

```diff
@@ -0,0 +1,104 @@
+#
```

Comment:
Many of the comments I made for `.github/docker/linux/x64/Dockerfile` apply to this file as well.